### ISSUE TITLE
Too slow when converting some ODS files

### ISSUE THREAD

**Issue by naoliv** [dstosberg/odt2txt#13]
Originally from https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=545551
 
Sometimes odt2txt gets too slow when converting some ODS files.
For example, [using this spreadsheet](https://bugs.debian.org/cgi-bin/bugreport.cgi?att=1;bug=545551;filename=dev_data_01.ods;msg=5) (it isn't exactly a big file: it has 16196 rows X 10 columns) it takes 14+ minutes to convert it.

Maybe this could be improved somehow?

**Comment by dstosberg** [comment]
Yes, this could probably be solved by a complete rewrite. The whole approach, which is to convert the document by using regular expressions, is flawed. It doesn't scale to large documents. What we would need, is a new tool using a proper XML parser.
